src/common/hooks/bankaccounts.ts

```diff
@@ -0,0 +1,13 @@
+import { endpoints } from 'common/api-endpoints'
+import { BankAccountResponse } from 'gql/bankaccounts'
```

Comment:
the automated build fails on this line, could you check if you have committed all needed files

Reply:
I checked it, everything seems correct. Not sure why its not happy with this import tbh.